pkg/kubelet/apis/kubeletconfig/v1beta1/defaults.go

```diff
@@ -63,118 +82,95 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 	if obj.Authorization.Webhook.CacheUnauthorizedTTL == zeroDuration {
 		obj.Authorization.Webhook.CacheUnauthorizedTTL = metav1.Duration{Duration: 30 * time.Second}
 	}
-	if obj.Address == "" {
-		obj.Address = "0.0.0.0"
-	}
-	if obj.VolumeStatsAggPeriod == zeroDuration {
-		obj.VolumeStatsAggPeriod = metav1.Duration{Duration: time.Minute}
+	if obj.RegistryPullQPS == nil {
+		obj.RegistryPullQPS = utilpointer.Int32Ptr(5)
 	}
-	if obj.RuntimeRequestTimeout == zeroDuration {
-		obj.RuntimeRequestTimeout = metav1.Duration{Duration: 2 * time.Minute}
+	if obj.RegistryBurst == 0 {
+		obj.RegistryBurst = 10
 	}
-	if obj.CPUCFSQuota == nil {
-		obj.CPUCFSQuota = boolVar(true)
+	if obj.EventRecordQPS == nil {
+		obj.EventRecordQPS = utilpointer.Int32Ptr(5)
 	}
 	if obj.EventBurst == 0 {
 		obj.EventBurst = 10
 	}
-	if obj.EventRecordQPS == nil {
-		temp := int32(5)
-		obj.EventRecordQPS = &temp
-	}
-	if obj.EnableControllerAttachDetach == nil {
-		obj.EnableControllerAttachDetach = boolVar(true)
-	}
 	if obj.EnableDebuggingHandlers == nil {
-		obj.EnableDebuggingHandlers = boolVar(true)
+		obj.EnableDebuggingHandlers = utilpointer.BoolPtr(true)
```

Comment:
"DebuggingHandlers" is poorly named. I would eventually like this to go away, in favor of using authorization controls for this purpose. If we really want hard controls for this, I'd rather see a per-handler policy, rather than this vague categorization.

I'm not sure there's any action needed in this PR, just FYI.